src/Services/RedisModule.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Autofac;
+using Lykke.Job.OrderBooksCacheProvider.Core;
+using StackExchange.Redis;
+
+namespace Lykke.Job.OrderBooksCacheProvider.Services
+{
+    public class RedisModule : Module
+    {
+        private readonly CacheSettings _settings;
+
+        public RedisModule(CacheSettings settings)
+        {
+            _settings = settings;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            var redis = ConnectionMultiplexer.Connect(_settings.RedisConfiguration);
```

Comment:
Can be registered via  builder.Register. Container registration is not the best time for open external connections